02-frequency_filter.py

```diff
@@ -92,8 +92,7 @@ def run_filter(subject, run=None, session=None):
         raw.plot(n_channels=50, butterfly=True)
 
         # plot power spectral densitiy
-        raw.plot_psd(area_mode='range', tmin=10.0, tmax=100.0,
-                     fmin=0., fmax=50., average=True)
+        raw.plot_psd(average=True, area_mode='range')
```

Comment:
I don't think it's a good idea to average. non-averaged plots are excellent ways to spot remaining bad channels

Reply:
Why not plot both? :)

Reply:
Also… the non-avg plot will take quite a while to be rendered :(

Reply:
Can you use average False here please ?